src/main/java/space/obminyashka/items_exchange/security/jwt/refresh/RefreshToken.java

```diff
@@ -0,0 +1,40 @@
+package space.obminyashka.items_exchange.security.jwt.refresh;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import space.obminyashka.items_exchange.model.User;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+public class RefreshToken {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @OneToOne
+    @JoinColumn(name = "user_id", referencedColumnName = "id")
+    private User user;
+
+    @Column(nullable = false, unique = true)
+    private String token;
+
+    @Column(name = "expiry_date", columnDefinition = "DATE", nullable = false)
+    private LocalDateTime expiryDate;
+
+    @Column(name = "created", columnDefinition = "DATE", nullable = false)
+    private LocalDateTime created;
+
+    public RefreshToken(User user, String token, LocalDateTime expiryDate) {
+        this.user = user;
+        this.token = token;
+        this.expiryDate = expiryDate;
+        this.created = LocalDateTime.now();
```

Comment:
For that purpose I'd created CustomDateTimeProvider which has to populate date values. You only have to annotate them with CreatedDate.

Reply:
Unfortunately, I do not understand what you mean(